x-pack/test/security_solution_endpoint_api_int/apis/transform_helper.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+
+export async function putTransform(getService: (serviceName: 'es') => Client, transformId: string) {
```

Comment:
It looks like we put this transform in to get the tests to pass, which is good.  However, I pulled this down and tried it out and the UI breaks without this transform.

Can we put the transform in in the interim before it's available via the package?  I don't want to UI to break indefinitely before there's a package available.

Let me know if I'm missing something.